### DISCUSSION TITLE
Detail view resourceMode

### DISCUSSION THREAD

**Discussion by k-rustemuly** [moonshine-software/moonshine#458] [Ideas]
В детальном просмотре item HasMany  в resourceMode появляется отдельно от блоков. Почему бы не делать возможность вывести внутри блоков, табов и т.д

![изображение](https://github.com/moonshine-software/moonshine/assets/16911003/454a0ab4-dd6a-4eae-8b20-84c47927ac5b)

**Comment by lee-to** [comment]
Будет такая возможность в муншайн 2.0
В первой ветке уже делать не будем
